### PR TITLE
IntegerIndexedElementSet should always indicate success. Ref: tc39/ecma262/pull/2210

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by rwaldron** [comment]
@rkirsling Please take a look!

File: test/built-ins/TypedArray/prototype/includes/detached-buffer-tointeger.js [hunk 1]
```diff
@@ -2,30 +2,36 @@
...
+  let isDetached = false;
+  function valueOf(){
     $DETACHBUFFER(sample.buffer);
+    isDetached = true;
     return 0;
   }
-  assert.throws(TypeError, function() {
-    sample.includes(0, {valueOf :detachAndReturnIndex});
-  });
+  sample.includes(0, {valueOf});
```
Comment (rkirsling) [inline test/built-ins/TypedArray/prototype/includes/detached-buffer-tointeger.js:35]: I suppose we should assert that this returns `false`?

**Review by rkirsling — APPROVED** [review]

File: test/built-ins/TypedArray/prototype/includes/detached-buffer-tointeger.js (hunk 1 above)
Comment (rwaldron) [reply test/built-ins/TypedArray/prototype/includes/detached-buffer-tointeger.js:35]: Sure, we can do that!

**Comment by rwaldron** [comment]
@rkirsling I've updated the test that you commented on, can you take another look?

**Review by rkirsling — APPROVED** [review]
Thanks!

**Pull request merged**